packages/features/ee/round-robin/roundRobinManualReassignment.ts

```diff
@@ -242,8 +274,10 @@ export const roundRobinManualReassignment = async ({
     newReferencesToCreate,
   });
 
+  const { cancellationReason, ...evtWithoutCancellationReason } = evt;
+
   // Send emails
-  await sendRoundRobinScheduledEmailsAndSMS(evt, [
+  await sendRoundRobinScheduledEmailsAndSMS(evtWithoutCancellationReason, [
```

Comment:
if `cancellationReason` is set, email scheduled email looks like that: 
<img width="680" alt="Screenshot 2024-10-22 at 12 51 10 AM" src="https://github.com/user-attachments/assets/88da2f8a-14a8-4572-9ef9-cf74cc1ba815">